lib/bootloader_pvm.pm

```diff
@@ -157,13 +157,20 @@ sub boot_hmc_pvm {
     assert_screen 'pvm-vterm-closed';
 
     # power off the machine if it's still running - and don't give it a 2nd chance
-    type_string "chsysstate -r lpar -m $hmc_machine_name -o shutdown --immed --id $lpar_id && echo 'LPAR SUCCESSFULLY SHUT DOWN'\n";
-    assert_screen [qw(pvm-poweroff-successful pvm-poweroff-not-running)], 180;
-
+    # sometimes lpar shutdown takes long time, we need to check it's state
+    type_string "chsysstate -r lpar -m $hmc_machine_name -o shutdown --immed --id $lpar_id && echo 'LPAR SHUTDOWN NOW'\n";
+    script_run("lssyscfg -m $hmc_machine_name -r lpar --filter \"\"lpar_ids=$lpar_id\"\" -F state");
+    assert_screen [qw(lpar-is-running lpar-is-down)];
+    if (match_has_tag('lpar-is-running')) {
+        record_info('LPAR is still running, waiting for shutdown');
+        script_run("for ((i=0\; i<10\; i++)) \; do lssyscfg -m $hmc_machine_name -r lpar --filter \"\"lpar_ids=$lpar_id\"\" -F state | grep -q \"Not Activated\" && echo \"LPAR IS DOWN\" && break || echo \"Waiting for lpar $lpar_id to shutdown\" && sleep 5 \; done");
```

Comment:
`script_run` relies on `wait_serial`, and you can clearly see [Cannot redirect output](https://openqa.suse.de/tests/4294064#step/bootloader/5) as a result of that. change it to type string.